### ISSUE TITLE
vaue error condition leads to "Cluster .. not found"

### ISSUE THREAD

**Issue by lopf** [Napsty/check_rancher2#24]
Hi!
https://github.com/Napsty/check_rancher2/blob/d93c4dee2b0b8a595538c128318535e4c9e27a0a/check_rancher2.sh#L225
the condition checks for the string "error" which could also be a configuration, e.g. for ingress:
```
"options": {
    "custom-http-errors": "404,503",
    "use-forwarded-headers": "true"
}
```
It's not clear to me what the initial intention was. Can you give an example so that I can improve the error string to not match valid options like `custom-http-errors`?

Thank you anyway for your check!

**Comment by Napsty** [comment]
Hi! 

Thx for reporting this. Indeed, the Nginx option "custom-http-errors" hits the grep ^^

The idea is that the plugin first verifies whether the given cluster exists or not. When using a non-existing cluster, the following error is given back by the Rancher API:

```
{"baseType":"error","code":"NotFound","message":"clusters.management.cattle.io \"c-qvf7pxxx\" not found","status":404,"type":"error"}
```

So we could change the string from "error" to "NotFound" which would be a better fit I suppose.

**Comment by Napsty** [comment]
fixed by #25

**Comment by Napsty** [comment]
There was another cluster not found grep further down in the plugin and the same logic (NotFound instead of error) was now also applied on the "project" check type.

See commit https://github.com/Napsty/check_rancher2/commit/79881ef67aaf39cfa0bc4b2d4eef3790636059a6
